configure.ac

```diff
@@ -338,7 +338,7 @@ AC_MSG_RESULT([
 
 -------------------------------------------------------------------------
 |    ___ _   _ ___                                                      |
-|   / __| | | |_  )   Release 7.2.0 'Blackbird'                         |
```

Comment:
Nice catch. Is there not an automatic procedure to update the version numbers?

Reply:
There is a procedure, but these files may get overlooked.